packages/PreviewCard/PreviewCard.md

```diff
@@ -0,0 +1,35 @@
+### Usage Criteria
+
+- Use when you want to showcase and guide your users to a webpage.
```

Comment:
@Andrew-K-Lam would it make sense to add some verbiage around using [ResponsiveImage](https://github.com/telus/redux-contentful#responsiveimage) since most users will get their images from _Contentful_ and adding a quick link to use _ResponsiveImage_ would be a nice touch.

Reply:
Contentful only returns the URL to the image, it is up to the user to decide what kind of image library (TDS-image, img tag) to use with `PreviewCard`.  `PreviewCard` could also be used outside of Contentful with another CMS, so I don't think it's necessary to add verbiage specific to ResponsiveImage.
For example, I will be using this component with Contentful, using a custom `<LazyLoadImage />` that wraps `react-simple-img` library.